src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,154 @@
+  }
+
+  // Footer
+  &.pf-m-footer {
+    margin-top: var(--pf-c-inline-edit__group--m-footer--MarginTop);
+  }
+
+  // Column
+  &.pf-m-column {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+
+    flex-direction: column;
+  }
+
+  // Enable groups
+  &.pf-m-enable {
+    display: inline-flex;
+    visibility: visible;
+
+    .pf-c-inline-edit__action {
+      display: inline-block;
+      visibility: visible;
+    }
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+  }
+}
+
+// Table action cells and action groups inside input groups
+.pf-c-inline-edit .pf-c-table__action,
+.pf-c-inline-edit__group.pf-m-input-group {
+  .pf-c-inline-edit__group.pf-m-action-group {
+    --pf-c-inline-edit__group--m-action-group--item--MarginRight: 0;
+    --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+  }
+}
+
+// Input and input group
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__group.pf-m-input-group {
+  flex: 1;
+}
+
+// Action
+.pf-c-inline-edit__action {
+  &.pf-m-icon-button {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__action--m-icon-button--MarginRight);
+  }
+
+  .pf-c-button.pf-m-plain.pf-m-valid {
+    --pf-c-button--m-plain--Color: var(--pf-c-inline-edit__action--c-button--m-valid--m-plain--Color);
+    --pf-c-button--m-plain--hover--Color: var(--pf-c-inline-edit__action--c-button--m-valid--m-plain--hover--Color);
+  }
+}
+
+// Input and input group
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__action,
+.pf-c-inline-edit__group.pf-m-input-group,
+.pf-c-inline-edit__group.pf-m-action-group {
+  display: none;
+  visibility: hidden;
+}
+
+// Editable
+.pf-c-inline-edit.pf-m-inline-editable,
+.pf-c-inline-edit .pf-m-inline-editable {
+  // Expose inputs, input groups, actions and action groups
+  .pf-c-inline-edit__input,
+  .pf-c-inline-edit__action,
+  .pf-c-inline-edit__group.pf-m-action-group,
+  .pf-c-inline-edit__group.pf-m-input-group {
+    visibility: visible;
+  }
+
+  // Input
+  .pf-c-inline-edit__input {
+    display: block;
+  }
+
+  // Action
+  .pf-c-inline-edit__action {
+    display: inline-block;
+  }
+
+  // Action group
+  .pf-c-inline-edit__group.pf-m-action-group {
+    display: inline-flex;
+  }
+
+  // Input group
+  .pf-c-inline-edit__group.pf-m-input-group {
+    display: flex;
+  }
+
+  // Hide value, value group and enable actions
+  .pf-c-inline-edit__value,
+  .pf-c-inline-edit__action.pf-m-enable,
+  .pf-c-inline-edit__group.pf-m-enable {
+    display: none;
+    visibility: hidden;
+  }
+}
+
+.pf-c-inline-edit__action,
+.pf-c-inline-edit__group {
+  &.pf-m-enable {
+    display: inline-block;
+    visibility: visible;
+  }
+}
+
+// offset button by top padding
+.pf-c-inline-edit__label + .pf-c-inline-edit__action.pf-m-enable > .pf-c-button {
+  margin-top: calc(var(--pf-c-button--PaddingTop) * -1);
+  margin-bottom: calc(var(--pf-c-button--PaddingBottom) * -1);
+}
+
+.pf-c-inline-edit__label.pf-m-bold {
```

Comment:
if this is still used/needed, can you add it to the documentation? I'm curious when it would be used.